ios/Gekidou/Sources/Gekidou/Storage/Database+Users.swift

```diff
@@ -119,6 +119,24 @@ extension Database {
         
         throw DatabaseError.NoResults(query.expression.description)
     }
+    
+    public func getUserLastPictureAt(for userId: String,  withServerUrl serverUrl: String) -> Double? {
+        let idCol = Expression<String>("id")
+        var updateAt: Double?
```

Comment:
Basically the `if let` statement here https://github.com/mattermost/mattermost-mobile/pull/7124/files#diff-3d347a0292e895f86709e0c59de6e8aaf0d46500c4a5ef73884c32dedeee8856R122 will make it so that in the case the value is null the if statement is false, thus not applying the value and the default value at the beginning of the function is being set to 0.